### DISCUSSION TITLE
No file import function for Integrations SDK?

### DISCUSSION THREAD

**Discussion by Seazonx** [jgraph/drawio#4276] [Q&A] [answered]
According to https://www.drawio.com/integrations
draw.io is integrated in Lark
But there's no file import function in Lark draw.io integration.
I've asked the Lark team, and they said that file import function is not included in draw.io SDK.
I wanna know is it true?
![20240318-150050](https://github.com/jgraph/drawio/assets/41646287/886a6509-32a2-43de-b286-44916c5875bd)

**Comment by davidjgraph** [comment]
We have no idea, the Lark team will know the answer best.

**Reply by Seazonx** [reply]
Thank you for reply
My question is,  **File Import** function is included in draw.io integration SDK or not?

**Reply by davidjgraph** [reply]
What is the "draw.io integration SDK"?

**Reply by Seazonx** [reply]
OK, if we integrate draw.io into our own product, will we have the File Import function like the draw.io online editor?
![20240318-170127](https://github.com/jgraph/drawio/assets/41646287/4e701ba1-f461-4e0b-a69f-33d8f668017a)

**Reply by davidjgraph** [reply, marked as answer]
This repo is the repo of the editor provided in the online version. If you deploy it you can effectively clone pretty much everything in the online version.

**Reply by Seazonx** [reply]
Thank You!
